guide/contribute/formatting.md

```diff
@@ -175,6 +175,36 @@ Let's start with a very wide image that extends beyond the content width on desk
    height = 800
 %}
 
+#### Image viewable in larger modal
+
+{% raw %}
+```liquid
+{% include picture.html
+   image = "/assets/images/guide/contribute/formatting/example-optional-image-modal.png"
+   retina = "/assets/images/guide/contribute/formatting/example-optional-image-modal@2x.png"
+   mobile = "/assets/images/guide/contribute/formatting/example-optional-image-modal-mobile.png"
+   mobileRetina = "/assets/images/guide/contribute/formatting/example-optional-image-modal-mobile@2x.png"
+   modalImage = "/assets/images/guide/contribute/formatting/example-optional-image-modal@2x.png"
+   modalCaption = "An optional caption to display with the image"
```

Comment:
The picture include already has a caption, curious what's the rationale of adding a specific parameter for modal caption. 👇  is a "caption" on the picture.html include

<img width="217" alt="Screenshot 2022-01-22 at 1 30 12 PM" src="https://user-images.githubusercontent.com/183140/150649200-2cff167f-cf20-4f4f-9856-95de02113fe6.png">

Reply:
There was some talk in #267 about being able to provide additional information with each image. But it could easily just re-use the same caption.

Reply:
@GBKS and @Bosch-0, any thoughts on this? Should there be a separate caption for modal images, or should it just use the same caption as the original image?

Reply:
I'd not introduce a new caption. Reason is that this would lead to content hidden from a "normal" page read-through.

Overall, I think this image zoom should not expose new information, but should make the information on the page (in the image) just easier to see.

Reply:
Agree with GBKS, a new caption seems unnecessary.